docs/items/interactionpipeline.md

```diff
@@ -0,0 +1,115 @@
+The Interaction Pipeline
```

Comment:
Again, I love the detail, but this is getting into Javadoc territory, where it probably would be better to put this into Parchment or NeoForge instead of having it here.

Reply:
While I agree that the method reference should be Javadocs, I put this in here on purpose because this isn't immediately clear just by looking at the code, even for the methods where Javadocs are already present. Finding out the exact order in which things are called requires looking through some fairly dense code in `MultiPlayerGameMode` and `Minecraft`, which the average modder shouldn't be required to understand in my opinion.

Reply:
Fair enough. It just felt like when I was reading it that it was a giant wall of text, making it hard to consume. I'm not sure what's best to do about this without losing the quality.